cmd/beacon-chain/flags/base.go

```diff
@@ -16,10 +16,10 @@ var (
 		Usage: "A MEV builder relay string http endpoint, this wil be used to interact MEV builder network using API defined in: https://ethereum.github.io/builder-specs/#/Builder",
 		Value: "",
 	}
-	// HTTPWeb3ProviderFlag provides an HTTP access endpoint to an ETH 1.0 RPC.
-	HTTPWeb3ProviderFlag = &cli.StringFlag{
-		Name:  "http-web3provider",
-		Usage: "A mainchain web3 provider string http endpoint. Can contain auth header as well in the format --http-web3provider=\"https://goerli.infura.io/v3/xxxx,Basic xxx\" for project secret (base64 encoded) and --http-web3provider=\"https://goerli.infura.io/v3/xxxx,Bearer xxx\" for jwt use",
+	// ExecutionEngineEndpoint provides an HTTP access endpoint to connect to an execution client on the execution layer
+	ExecutionEngineEndpoint = &cli.StringFlag{
+		Name:  "execution-endpoint",
+		Usage: "An execution client http endpoint. Can contain auth header as well in the format",
```

Comment:
If you add an alias for the old flag name, then this isn't really a breaking change.

Reply:
Good pt. I added the alias but updated the default port to 8551 since 8545 is no longer relevant post merge. Due to port changes, it's breaking now